vsandroid_packaging.lua

```diff
@@ -243,7 +243,7 @@
 	end
 	
 	function pack.androidAppLibName(cfg)
-		vc2010.element("AndroidAppLibName", nil, "$(RootNamespace)")	-- TODO: fix
+		vc2010.element("AndroidAppLibName", nil, cfg.androidapplibname)
```

Comment:
... but since Android is not a platform, I am well familiar, I'd rather allow it to have one name for all configs, so this should be prj.androidapplibname, instead of cfg